pepper-apis/src/main/java/org/broadinstitute/ddp/model/kit/KitConfiguration.java

```diff
@@ -41,28 +44,44 @@ public String getStudyGuid() {
         return studyGuid;
     }
 
+    public Collection<KitRule> getRules() {
+        return List.copyOf(rules);
+    }
+
     public boolean evaluate(Handle handle, String userGuid) {
```

Comment:
This method is called as part of the kit check/creation process in Housekeeping. The new code here should do what we need -- that is, _not_ creating a new kit if zip code doesn't match.